.github/workflows/cmx-simple.yaml

```diff
@@ -0,0 +1,84 @@
+name: Compatibilty Test with CMX in simple mode
```

Comment:
Seems like the primary difference here is how we install the charts. I don't have a preference how we do it between the two options you've shown, but I do think we should collapse both of these into a single pipeline instead of having two of them for this reason.

Reply:
Good catch. let's keep it as a single pipeline.